lib/Sema/CalleeCandidateInfo.h

```diff
@@ -59,7 +59,7 @@ namespace swift {
   ///
   struct UncurriedCandidate {
```

Comment:
Perhaps worth renaming this too :)

Reply:
Are you thinking just to `Candidate`? :)

Reply:
OverloadCandidate maybe

Reply:
Sounds good! I'll do that in this PR and cleanup documentation around this.